app/bundles/FormBundle/Helper/FormUploader.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\FormBundle\Helper;
+
+use Mautic\CoreBundle\Exception\FileUploadException;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\FileUploader;
+use Mautic\FormBundle\Crate\UploadFileCrate;
+use Mautic\FormBundle\Entity\Field;
+use Mautic\FormBundle\Entity\Submission;
+
+class FormUploader
+{
+    /**
+     * @var FileUploader
+     */
+    private $fileUploader;
+
+    /**
+     * @var CoreParametersHelper
+     */
+    private $coreParametersHelper;
+
+    public function __construct(FileUploader $fileUploader, CoreParametersHelper $coreParametersHelper)
+    {
+        $this->fileUploader         = $fileUploader;
+        $this->coreParametersHelper = $coreParametersHelper;
+    }
+
+    /**
+     * @param UploadFileCrate $filesToUpload
+     * @param Submission      $submission
+     *
+     * @throws FileUploadException
+     */
+    public function uploadFiles(UploadFileCrate $filesToUpload, Submission $submission)
```

Comment:
Can we organize the files into form ID named directories to make it easier to clean up files when a form is deleted? Then maybe in FormBundle/Model/FormModel:deleteEntity and deleteEntities, we can simply wipe out that directory to quickly cleanup.